kerberos-environment/files/setup-client.ps1

```diff
@@ -43,6 +40,19 @@ New-ItemProperty -Path "HKLM:\Software\Policies\Google\Chrome" -PropertyType Str
 
 Write-Host  "DONE: applying Chrome Group Policy for Kerberos authentication"
 
+Write-Host  "Start: chromedriver"
+Invoke-WebRequest "https://chromedriver.storage.googleapis.com/79.0.3945.36/chromedriver_win32.zip" -OutFile $env:TEMP\chromedriver_win32.zip; 
+Expand-Archive -LiteralPath $env:TEMP\chromedriver_win32.zip -DestinationPath C:\chromedriver
+Remove-Item $env:TEMP\chromedriver_win32.zip;
+[System.Environment]::SetEnvironmentVariable("PATH", $Env:Path + ";C:\chromedriver", "Machine")
+Write-Host  "DONE: chromedriver installation"
+
+# install package manager
+Set-ExecutionPolicy Bypass -Scope Process -Force; Invoke-Expression ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))
+
+# install tools
+choco install jdk8 --force -y -and choco install git --force -y -and choco install maven --force -y
```

Comment:
Why JDK8?

Reply:
because we are using 1.8 [pom.xml](https://git.alfresco.com/platform-services/alfresco-dbp-test/blob/master/pom.xml#L16) . I will change it to java11 as we talked offline ;)